### PR TITLE
Ls 447 prod cluster

### PR DESCRIPTION
**Pull request by mendonk**
* CDC enabled pulsar recommendations
* Helm production recs from rodrigo

### CODE REVIEW

File: modules/ROOT/pages/production-cluster.adoc [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+Our recommendations are based on the following preconditions: +
+
+* C* replication factor for all CDC tables: 3 +
+* Pulsar message replication factor: 3 +
+* Pulsar message retention and TTL period is *3 hours*. All *acked* or *unacked* CDC messages will be kept in the Pulsar cluster for one day. This means: +
+** Any unacknowledged messages will be deleted after being kept in the cluster for *3 hours* (TTL period) +
+** Any acknowledged messages will continue staying in the cluster for up to *3 hours* (retention period) +
+
+.Pulsar Kubernetes cluster node specifications
```
Comment (yabinmeng) [inline modules/ROOT/pages/production-cluster.adoc:19]: Again, double check this with engineering.

File: modules/ROOT/pages/production-cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+|Name
+|Description
+|Instance Type
+|vCPU
+|RAM
+|Node count
+
+|Pulsar-group
+|Managed node group dedicated to Pulsar components (Pulsar proxies, zookeepers, brokers, and bookkeepers)
+|m6a.2xlarge
```
Comment (yabinmeng) [inline modules/ROOT/pages/production-cluster.adoc:31]: This is only relevant to AWS. You may want to state this fact.

**Review by yabinmeng — CHANGES_REQUESTED** [review]
Since production recommendation is an important topic, I think we need to be a little bit more thorough with this document. 

Also I don't think it is needed to put capacity planning as part of the production recommendation doc for general public usage. The customer oriented deployment recommendation is not suitable to be put as part of the general usage document.

If you have questions, let's set up a meeting to discuss.

**Comment by mendonk** [comment]
Superseded by https://github.com/datastax/pulsar-docs/pull/105

**Pull request closed without merging**